src/components/home.js

```diff
@@ -40,24 +40,32 @@ function Home(props) {
   const getStates = async () => {
     try {
       const [
-        response,
+        {data},
         stateDistrictWiseResponse,
         {data: statesDailyResponse},
         updateLogResponse,
-        stateTestResponse,
+        {data: stateTestData},
       ] = await Promise.all([
```

Comment:
use Promise.allSettled() 🤔

Reply:
any specific reason?

Reply:
If this recommendation is regarding the null check you mentioned, please create an issue. If accepted you can fix it and submit a PR.